org_eclipse_uprotocol/cloudevent/datamodel/ucloudeventattributes.py

```diff
@@ -0,0 +1,175 @@
+# -------------------------------------------------------------------------
+
+# Copyright (c) 2023 General Motors GTO LLC
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# SPDX-FileType: SOURCE
+# SPDX-FileCopyrightText: 2023 General Motors GTO LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# -------------------------------------------------------------------------
+
+from typing import Optional
+
+from org_eclipse_uprotocol.proto.uattributes_pb2 import UPriority
+from org_eclipse_uprotocol.proto.uattributes_pb2 import UPriority
+
+
+class UCloudEventAttributes:
+    """
+    Specifies the properties that can configure the UCloudEvent.
+    """
+
+    def __init__(self, hash_value: str, priority: UPriority, ttl: int, token: str):
+        """
+        Construct the properties object.<br><br>
+        @param hash_value: an HMAC generated on the data portion of the CloudEvent message using the device key.
+        @param priority: uProtocol Prioritization classifications defined at QoS in SDV-202.
```

Comment:
Please remove all references to "SDV-202" as that was my internal specification document name